core/optaplanner-core-impl/src/main/java/org/optaplanner/core/impl/score/director/AbstractScoreDirector.java

```diff
@@ -752,12 +754,18 @@ private Map<Object, Set<ConstraintMatch<Score_>>> createConstraintMatchMap(
         for (ConstraintMatchTotal<Score_> constraintMatchTotal : constraintMatchTotals) {
             String constraintId = constraintMatchTotal.getConstraintId();
             for (ConstraintMatch<Score_> constraintMatch : constraintMatchTotal.getConstraintMatchSet()) {
-                Stream.Builder<Object> keyStream = Stream.builder().add(constraintId);
-                // The order of justificationLists for constraints that include accumulates isn't stable, so we make it.
-                constraintMatch.getJustificationList()
-                        .stream()
-                        .sorted(solutionDescriptor.getClassAndPlanningIdComparator())
-                        .forEach(keyStream);
+                Stream.Builder<Object> keyStream = Stream.builder()
```

Comment:
Sonar suggests this code is not covered by tests; so is the entire `buildScoreCorruptionAnalysis()`, which calls this code.